extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -373,6 +377,18 @@ export class AmpStoryEmbeddedComponent {
     }
   }
 
+  /**
+   * Schedules embeds to be paused.
+   * @param {!Element} embedEl
+   * @private
+   */
+  scheduleEmbedToPause_(embedEl) {
+    this.resources_.scheduleResume(this.storyEl_, embedEl);
```

Comment:
Sorry, why do we want to schedule resume on it when we're try to schedule pause?  (there might be a good reason, but the naming here is unintuitive, so a comment would be great)

Reply:
We must call `scheduleResume` on anyone that has called `schedulePause`, otherwise the resource would remain unloaded. 

It will also have no effect on elements that haven't been paused, it will only call `resumeCallback` on elements that have been previously paused.

See: https://github.com/ampproject/amphtml/blob/1d53d2e89306a22411a03a8f16375c6d7d287ffb/src/service/resources-impl.js#L772-L779

Reply:
Can you just actually leave that as a comment inline?

Reply:
Whoops, sorry.  I see that you did that 😄